hyper/contrib.py

```diff
@@ -19,6 +19,10 @@
 from hyper.common.connection import HTTPConnection
 from hyper.compat import urlparse, ssl
 from hyper.tls import init_context
+from sys import version_info as pyver
+
+
+conditionalToStr=lambda s: s.decode("utf-8") if pyver > 2 else lambda s: s
```

Comment:
Absolutely no need to use a lambda here, let alone two: just use [this function](https://github.com/Lukasa/hyper/blob/development/hyper/common/util.py#L57-L61).